analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -348,14 +348,14 @@ class Pip(
         val jsonFile = createOrtTempDir().resolve("python-inspector.json")
 
         val pythonInspector = if (definitionFile.name == "setup.py") {
-            PythonInspector.runPythonInspector(
+            PythonInspector.run(
```

Comment:
out of scope: Can `PythonInspector`  be dropped from `PythonInspector.run`?

Reply:
No, the call comes from another class.